kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/config/CoverageConfig.kt

```diff
@@ -0,0 +1,25 @@
+package kotlinx.fuzz.config
+
+interface CoverageConfig {
+    val reportTypes: Set<CoverageReportType>
+    val includeDependencies: Set<String>
+}
+
+class CoverageConfigImpl internal constructor(builder: KFuzzConfigBuilder) : CoverageConfig {
+    override var reportTypes by builder.KFuzzPropProvider<Set<CoverageReportType>>(
+        nameSuffix = "coverage.reportTypes",
```

Comment:
I think that it might be useful to expand name prefix so that we don't need to copy-paste `coverage.`